src/components/systemButtons/systemButtons.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { cva, VariantProps } from 'class-variance-authority';
+
+const buttonVariants = cva(
+  [
+    'rounded-lg flex items-center justify-center cursor-pointer',
+    'transition-all duration-300 ease-in-out focus:outline-none',
+  ],
+  {
+    variants: {
+      size: {
+        large: 'w-[170px] h-[56px]',
+        medium: 'w-[170px] h-[42px]',
+        // small: '',
+      },
+      variant:{
+        filled: {
+            large: 'bg-primary-400',
+            medium: 'bg-primary-400',
+            // small: '',
+        },
+        outline: {
+            large: 'border border-primary-400',
+            medium: 'border border-primary-400',
+            // small: '',
+        },
+        ghost: {
+            large: 'bg-transparent',
+            medium: 'bg-transparent',
+            // small: ''
+        },
+      },
+      state: {
+        default: 'bg-primary-400',
+        ghostdefault: 'bg-transparent',
+        ghosthover: 'bg-gray-400',
+        defaultoutline: 'bg-transparent border-2 border-primary-400',
+        hover: 'bg-primary-500',
+        hoveroutline: 'bg-gray-400 border-2 border-primary-400',
+        disabled: 'bg-gray-200 cursor-not-allowed',
+      },
+      typography:{
+        filleddefault: 'font-inter text-black font-medium leading-[19px] tracking-[0em] text-left',
```

Comment:
Only set the color of the text for all the typographies, delete all the other things (font-inter text-black font-medium leading-[19px] tracking-[0em] text-left).
Also, you shouldn't have to set the font, the leading or the tracking nowhere in the app. You just use the styles provided in the typhograpies of the app (<Body1>,<Body2>, etc)